dangerfile.ts

```diff
@@ -4,6 +4,7 @@ import path = require("path");
 import cp = require("child_process");
 import { suggestionsDir } from "@definitelytyped/utils";
 import { danger, fail, markdown } from "danger";
+import { getDTName } from "dts-gen/dist/names.js";
```

Comment:
This should actually be `unmangleScopedPackage`, I think.

Reply:
Or `typesPackageNameToRealName`, or similar; I believe we usually precheck in the dt-tools codebase.

But, definitely not `dts-gen` as a dep. It should be treated as a CLI tool only.

Reply:
ok - happy to update it, i just want to have a single source of truth, ideally for going in both directions.

Reply:
which package has `unmangleScopedPackage` or `typesPackageNameToRealName`?